### ISSUE TITLE
Implement hide/show for Controller?

### ISSUE THREAD

**Issue by Peque** [georgealways/lil-gui#54]
I am currently using this snippet:

```js
Controller.prototype.hide = function() {                                                                              
    this.domElement.style.display = "none";
};
Controller.prototype.show = function() {
    this.domElement.style.display = "";
};
```

But, since hide/show is already implemented for GUI, maybe it would make sense to add it for Controller too? :blush:

**Comment by georgealways** [comment]
Yea, I don't see why not! I'll put it in the todo list, but feel free to take a stab at a PR.

**Comment by Peque** [comment]
@georgealways That's the least I can do! :blush:  (https://github.com/georgealways/lil-gui/pull/55)

**Comment by georgealways** [comment]
Yeaaa, didn't mean for that to go in master. I made a new PR for that (#60) and that's merged into dev now. But you're right, this should stay open until release!

**Comment by georgealways** [comment]
Fixed in 0.17.0
